docs/conan.md

```diff
@@ -50,6 +54,7 @@ The highlighted parts can be adjusted:
 - ***conan-generated***: directory (absolute or relative) where the generated files will appear. This value is used in CMake presets from VCMI, but you can actually use any directory and override it in your local CMake presets.
 - ***never***: use this value to avoid building any dependency from source. You can also use `missing` to build recipes, that are not present in your local cache, from source.
 - ***CI/conan/PROFILE***: if you want to consume our prebuilt binaries, ***PROFILE*** must be replaced with one of filenames from our [Conan profiles directory](../CI/conan) (determining the right file should be straight-forward). Otherwise, either select one of our profiles or replace ***CI/conan/PROFILE*** with `default` (your default profile).
+- ***note for Windows x86***: we do not have prebuilt profile for Windows x86, but if you replace ***arch=x86_64*** to ***arch=x86*** and ***target_host=x86_64-w64-mingw32*** to ***target_host=i686-w64-mingw32*** then it will be working profile for 32-bit windows.
```

Comment:
there's no need to edit profile, it's possible to customise settings on the command line: should be `-s arch=x86` if I'm not mistaken. Please check if `target_host` can also be overridden from command line.

alternatively, create a "base" profile with common parts that will be included into dedicated 32- and 64-bit ones, see Apple profiles for an example.

and adjust the example in docs accordingly.

Reply:
@kambala-decapitator
1. I cannot move deps to main repo (no write rights in VCMI repo), if you move it, I will point script and doc to new location.
4. No, it is not possible, ffmpeg unconditionally required mfplat for Windows.

target_host cannot be overriden from command line in current context. Base profile is also cannot be created, because $target_host cannot be passed into included profile. I can try to make a profile as Jinja2 template, but later today.

Reply:
@kambala-decapitator on Conan. I created an issue in CCI for adding option to building FFmpeg without mediafoundation.

Reply:
Done 2 profiles and doc changes accordingly